### PR TITLE
Volumetric support for MeasureTexture

### LINKED ISSUE #2454: MeasureTexture should support three-dimensional images

### PR DESCRIPTION
**Pull request by mcquin**
Resolves #2454

### CODE REVIEW

File: cellprofiler/modules/measuretexture.py [hunk 1]
```diff
@@ -710,201 +539,111 @@ def display(self, workspace, figure):
...
+                if image.has_mask:
+                    mask, m2 = cellprofiler.object.size_similarly(labels, image.mask)
+                    mask[~m2] = False
                 else:
-                    mask, m2 = cellprofiler.object.size_similarly(labels, mask)
+                    mask = m1
 
-                    mask[~m2] = False
+        pixel_data[~mask] = 0
+        # mahotas.features.haralick bricks itself when provided a dtype larger than uint8 (version 1.4.3)
```
Comment (0x00b1) [inline cellprofiler/modules/measuretexture.py]: Is this true regardless of your available system memory? http://mahotas.readthedocs.io/en/latest/faq.html#i-ran-out-of-memory-computing-haralick-features-on-16-bit-images-is-it-not-supported

**Comment by mcquin** [comment]
In the issue, Anne suggests it's fine to drop Gabor.

File: cellprofiler/modules/measuretexture.py (hunk 1 above)
Comment (0x00b1) [inline cellprofiler/modules/measuretexture.py]: You could also use scikit-image to compute the Haralick features (from the GLCM).

Comment (mcquin) [inline cellprofiler/modules/measuretexture.py]: How much memory is required is unclear. However, I will take the 12-bit range allowance into consideration.

Thanks for the skimage suggestion. The mahotas function is much easier to use. I'll stick with it.

Comment (0x00b1) [inline cellprofiler/modules/measuretexture.py]: > How much memory is required is unclear. However, I will take the 12-bit range allowance into consideration.

If I’m reading the FAQ correctly, it’s `(ngrey, ngrey)` so 16-bit is `(2^16 − 1, 2^16 − 1)`.

**Comment by 0x00b1** [comment]
Nice work dude. I be this feels good. 👍

**Pull request merged**